src/js/nav-collapsible.js

```diff
@@ -0,0 +1,30 @@
+export function nav() {
+  "use strict";
```

Comment:
Webpack will take care of modularity - since this file doesn't actually export anything, you can just write your code. Webpack adds `"use strict"`, too.